### PR TITLE
Chapter1/homework/5

### PR DESCRIPTION
**Pull request by daman92**
I tried the null check two different ways with no success. Any guidance appreciated. Other wise, all tests complete.

### CODE REVIEW

**Comment by daman92** [comment]
In the project all tests now pass. Not sure why Codacy and Azure pipelines keeps failing. Thanks for the tip on the null check.

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -29,8 +60,19 @@ public static void Reverse(int[] array)
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:61]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -39,8 +81,19 @@ public static int[] RemoveLast(int[] array)
         /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveFirst(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:77]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 3]
```diff
@@ -63,8 +138,23 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:129]: Reuse insertAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 4]
```diff
@@ -75,8 +165,23 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:149]: Reuse insertAt

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 1 above)
Comment (daman92) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:61]: implemented

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 2 above)
Comment (daman92) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:77]: implemented

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 3 above)
Comment (daman92) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:129]: implemented

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 4 above)
Comment (daman92) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:149]: implemented

**Review by Almantask — APPROVED** [review]
Well done

**Pull request closed without merging**